src/uint/div.rs

```diff
@@ -17,35 +17,29 @@ impl<const LIMBS: usize> UInt<LIMBS> {
     /// When used with a fixed `rhs`, this function is constant-time with respect
     /// to `self`.
     pub(crate) const fn ct_div_rem(&self, rhs: &Self) -> (Self, Self, u8) {
-        let mut bd = self.bits().saturating_sub(rhs.bits());
+        let mb = rhs.bits();
+        let mut bd = (LIMBS * Limb::BIT_SIZE) - mb;
         let mut rem = *self;
         let mut quo = Self::ZERO;
-
         let mut c = rhs.shl_vartime(bd);
-        let mut e = Self::ONE.shl_vartime(bd);
 
         loop {
-            let mut r: Self = rem.wrapping_sub(&c);
-            let d = -(((1 - (r.limbs[LIMBS - 1].0 >> HI_BIT)) & 1) as SignedWord);
-            let d = d as Word;
-            rem = Self::ct_select(rem, r, d);
-            r = quo;
-            r = r.wrapping_add(&e);
-            quo = Self::ct_select(quo, r, d);
+            let (mut r, borrow) = rem.sbb(&c, Limb::ZERO);
+            rem = Self::ct_select(r, rem, borrow.0);
+            r = quo.bitor(&Self::ONE);
+            quo = Self::ct_select(r, quo, borrow.0);
             if bd == 0 {
                 break;
             }
             bd -= 1;
             c = c.shr_vartime(1);
-            e = e.shr_vartime(1);
+            quo = quo.shl_vartime(1);
+        }
+
+        let is_some = if mb == 0 { 0 } else { 1 };
```

Comment:
```suggestion
        let is_some = ((mb | -mb) >> 63) + 1;
```

Reply:
Or

```suggestion
        let is_some = Limb::is_zero(mb);
```

Reply:
Wouldn't `63` need to be `(Limb::BIT_SIZE - 1)`?

This seems ok but I'm also ok with a branch on a value computed from `rhs` where we already assume timing is variable in respect to its value